client/lib/reader-lists-items/store.js

```diff
@@ -0,0 +1,91 @@
+// Reader Lists Items Store
+
+// External dependencies
+import { List, fromJS } from 'immutable';
+import debugModule from 'debug';
+
+// Internal dependencies
+import { action as actionTypes } from './constants';
+import { createReducerStore } from 'lib/store';
+
+const debug = debugModule( 'calypso:reader-lists-items' ); //eslint-disable-line no-unused-vars
+
+const initialState = {
+	items: {},
+	errors: [],
+	currentPage: {},
+	isLastPage: {},
+	isFetching: false
+};
```

Comment:
I wonder if we should change this to be one Map of List ID -> List Details instead of three maps and one isFetching? So a `Map< ListID, Map< string, any > >` where the inner map holds `items`, `currentPage`, `isLastPage`, `isFetching`.